### ISSUE TITLE
Reply to a Message | BE

### ISSUE THREAD

**Issue by srmyers17** [department-of-veterans-affairs/va.gov-team#16270]
### **User Story**
As a user, I want to be able to respond to a message that has been sent to me so that I can communicate back-and-forth with my healthcare team. 

### **Acceptance Criteria**
- Verify functionality that users can respond to an existing message in a folder
- What time limits exist if the message is too old to respond to?
- Confirm that users cannot reply to a message in the delete folder? (assuming we display it) 

### **Designs**

**Comment by ashley-matthews** [comment]
User Story
As a user, I want to be able to respond to a message that has been sent to me so that I can communicate back-and-forth with my healthcare team. 

Acceptance Criteria
User has the ability to respond to a message

Tech Notes
Designs

**Comment by narin** [comment]
Message replies along with attachments are now working as expected. 

> What time limits exist if the message is too old to respond to?

* Need to verify if such a time limit exists AC with Barry.  I found this error message in the MHV docs that might be related: `130 - Unable to reply because the source message is expired`

> Confirm that users cannot reply to a message in the delete folder? (assuming we display it)

* I tested the API and MHV allows the user to reply to a message in the Deleted folder.  It would be up to the mobile client or vets-api to restrict this functionality.  It would make more sense to do so on the front end.
